pkg/channel/fanout/fanout_event_handler.go

```diff
@@ -60,8 +60,11 @@ type Subscription struct {
 // Config for a fanout.EventHandler.
 type Config struct {
 	Subscriptions []Subscription `json:"subscriptions"`
-	// Deprecated: AsyncHandler controls whether the Subscriptions are called synchronous or asynchronously.
```

Comment:
should we release note that this is _not_ `deprecated` anymore?

Reply:
This is only at the code level / library and we usually don't release note it since pretty much all users don't care